packages/workflow/src/workflow.ts

```diff
@@ -330,6 +345,7 @@ export async function scheduleLocalActivity<R>(
         seq,
         attempt,
         originalScheduleTime,
+        ...(summary !== undefined && { cmdOpts: { userMetadata: { summary } } }),
```

Comment:
Left over? I don't see any other traces of `cmdOpts`.

Reply:
yup, removed